packages/oauth/src/index.ts

```diff
@@ -455,93 +456,9 @@ export interface InstallationStore {
     installation: Installation<AuthVersion, boolean>,
     logger?: Logger): Promise<void>;
   fetchInstallation:
-    (query: InstallationQuery<boolean>, logger?: Logger) => Promise<Installation<'v1' | 'v2', boolean>>;
+  (query: InstallationQuery<boolean>, logger?: Logger) => Promise<Installation<'v1' | 'v2', boolean>>;
   deleteInstallation:
-    (query: InstallationQuery<boolean>, logger?: Logger) => Promise<void>;
-}
-
-// using a javascript object as a makeshift database for development
```

Comment:
Never fear, this code has just been relocated to `/stores/memory-store.ts` below. 👇🏼